Project-Service implementation-Script/common_config/config.ini

```diff
@@ -0,0 +1,133 @@
+[pre-prod]
+host = http://localhost:3569/
```

Comment:
I think this folder is not required. 
@MallanagoudaB

Reply:
@adithyadinesh0412  sorry but its a old file the new file is config.sample please check